### DISCUSSION TITLE
Penalizing the NER model in spaCy 3.0 for wrong predictions

### DISCUSSION THREAD

**Discussion by davesuketu215** [explosion/spaCy#7191] [Help: Coding & Implementations] [answered]
How can we imply a penalty on a custom NER if it makes a wrong prediction on the validation set?

**Comment by thomashacker** [comment, marked as answer]
Generally, the validation set is used to get the model's performance and not for adjusting. The **penalty** which you're referring to already happens in the training process by calculating the **loss** between the prediction and expected value
